app/classifier/tasks/dropdown/dropdown-dialog.cjsx

```diff
@@ -97,6 +105,46 @@ DropdownDialog = React.createClass
       deletedValues.push value
       @setState deletedValues: deletedValues
 
+  handlePresetChange: (e) ->
+    if e.target.value is "numberRange"
+      @setState numberRange: true
+    else
+      @setState numberRange: false
+
+  handlePresetApply: ->
+    select = @state.editSelect
+
+    if select.condition? and not @state.optionsKeys[select.condition]
+      return window.alert('Please select an answer to the related conditional dropdown(s) to associate the new option to')
+
+    optionsKey = @getOptionsKey(select)
+    preset = @refs.preset.value
+
+    switch preset
+      when "numberRange" then @handleNumberRange(select, optionsKey)
+      when "months" then select.options["#{optionsKey}"] = Months
+      when "Countries" then select.options["#{optionsKey}"] = Countries
+      when "statesUSA" then select.options["#{optionsKey}"] = statesUSA
+      when "provincesCanada" then select.options["#{optionsKey}"] = provincesCanada
+      when "statesMexico" then select.options["#{optionsKey}"] = statesMexico
+      else return
+
+    @setState editSelect: select
+    @setState numberRange: false
+    @refs.preset.value = ""
+
+  handleNumberRange: (select, optionsKey) ->
+    topNumber = @refs.topNumber.value
+    bottomNumber = @refs.bottomNumber.value
+
+    options = []
+    for num in [topNumber..bottomNumber]
```

Comment:
The `value` of an HTML input is always a string, so this is a range between two strings. It works because JavaScript coerces the values into numbers, but it's prettyyy weird. Maybe `parseFloat` those values first.